src/utilities/logging/Logger.cpp

```diff
@@ -410,9 +407,9 @@ void Logger::ManageLoggingEnvVars(bool set) {
                 // This is important when running calibrations since iterative runs of ngen
                 // can number in the 1000's and it is only necessary to retain the last ngen run
                 envVar = hydroModule + "_LOGFILEPATH";
```

Comment:
I think the subsequent change is potentially incorrect, because it's going to try to manipulate environment variables with dashes in their names, based on the various cases for `hydroModule`'s value above around line 374. The only characters guaranteed to work in environment variable names across applications, shells, libraries, etc are lower- and upper-case letters, digits, and underscore `_`. The conversion from `hydroModule` to `moduleNameInLog` seems to apply exactly that.

If there's a mis-match with the variable names various modules examine, we should fix the modules to use matching conservative names.